Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java

```diff
@@ -213,8 +213,8 @@ public static boolean isAllowAccessFromLocalMachine(final String subscriptionId,
         }
 
         public static boolean isAllowAccessFromLocalMachine(final List<FirewallRuleInner> firewallRules) {
```

Comment:
it's weird that we have to pass `firewallRules` as the parameter. I naturally think we should pass `server` as the parameter.
e.g. `isLocalMachineAllowedAccess(server)`

Reply:
isLocalMachineAllowedAccess(server) existed as well. it is a polymorphic one. in order to void to list firewall rules twice when when need to calculate access to local PC and other azure services.

Reply:
then this method may be renamed to something like `isLocalMachineIncluded(rules)`